### PR TITLE
code of conduct beta

### PR DESCRIPTION
**Pull request by LucaMele**
Code od conduct

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -14,24 +14,24 @@ We document everything that is relevant for other developers
...
 
-We aim to be as efficient as possible. On a PR review, we try to explain before referring to specs as we assume everyone have read the specs but maybe not understood it. We use link to specs as a nice add-on, not as a minimum argument. Of course, if explanation requires lots of time, just referring to specs might be ok. As a rule of thumb, always choose the option that solves the PR creators issue as soon as possible, as we have as goal to merge a PR as quick as possible. We also assume that everyone is willing to learn. Teaching is very welcome but is based on good will.
+**We aim to be as efficient as possible.** On a PR review, we try to explain before referring to specs as we assume everyone have read the specs but maybe not understood it. We use link to specs as a nice add-on, not as a minimum argument. Of course, if explanation requires lots of time, just referring to specs might be ok. As a rule of thumb, always choose the option that solves the PR creators issue as soon as possible, as we have as goal to merge a PR as quick as possible. We also assume that everyone is willing to learn. Teaching is very welcome but is based on good will.
```
Comment (AndyOGo) [inline CODE_OF_CONDUCT.md]: I think it's not only about refering to specs, but rather documentations, tutorials, articles, etc..
There other great resources online, like MDN, Medium, CSSTricks, CSSWizardy, Google Devs, etc.
I appreciate these a lot and want to use and reference them.

Comment (LucaMele) [reply CODE_OF_CONDUCT.md]: the idea is use links always as an add-on not as the only one (of course where it make sense and if u have time)

Comment (AndyOGo) [reply CODE_OF_CONDUCT.md]: But how about code examples? or reproduction repos?
These are not covered but very helpfull.

Comment (AndyOGo) [reply CODE_OF_CONDUCT.md]: gists, codesandbox, etc

Comment (LucaMele) [reply CODE_OF_CONDUCT.md]: I think the sentence: "As a rule of thumb, always choose the option that solves the PR creators issue as soon as possible, as we have as goal to merge a PR as quick as possible." covers it and also includes what u said. Or not?

Comment (AndyOGo) [reply CODE_OF_CONDUCT.md]: Okay

**Pull request merged**